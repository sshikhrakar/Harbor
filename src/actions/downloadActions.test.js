import {
  startDownload,
  completeDownload,
  updateCurrentDownloadProgress,
} from './downloadActions';

describe('ACTIONS: DOWNLOAD', () => {

  it('should create an action to  start a download', () =>  {
    expect(startDownload()).toMatchSnapshot();
  });

  it('should create an action to stop a download', () =>  {
    expect(completeDownload()).toMatchSnapshot();
  });


  it('should update the progress for current download', () => {
    expect(updateCurrentDownloadProgress(50)).toMatchSnapshot();
  });

});
