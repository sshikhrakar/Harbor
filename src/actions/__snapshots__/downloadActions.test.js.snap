// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ACTIONS: DOWNLOAD should create an action to  start a download 1`] = `
Object {
  "payload": Object {
    "project": undefined,
  },
  "type": "DOWNLOAD/DOWNLOAD_STARTED",
}
`;

exports[`ACTIONS: DOWNLOAD should create an action to stop a download 1`] = `
Object {
  "payload": Object {
    "project": undefined,
  },
  "type": "DOWNLOAD/DOWNLOAD_COMPLETE",
}
`;

exports[`ACTIONS: DOWNLOAD should update the progress for current download 1`] = `
Object {
  "payload": Object {
    "progress": 50,
  },
  "type": "DOWNLOAD/UPDATE_CURRENT_DOWNLOAD_PROGRESS",
}
`;
